apps/condo/domains/organization/schema/Organization.test.js

```diff
@@ -30,20 +28,31 @@ describe('Organization', () => {
         expect(data).toEqual({ meta: { count: null } })
     })
 
-    test('user: allow to getAll', async () => {
+    test('user: can read only organizations, it employed in', async () => {
         const admin = await makeLoggedInAdminClient()
-        await createOrganization(admin)
-        const [, userAttrs] = await createTestUser(admin)
-        const client = await makeLoggedInClient(userAttrs)
+        await createTestOrganization(admin)
+        const client = await makeClientWithRegisteredOrganization()
+
         const objs = await Organization.getAll(client, {})
-        expect(objs.length).toBeGreaterThan(0)
+        expect(objs.length).toBe(1)
+    })
+
+    test('admin: can read all organizations', async () => {
+        const admin = await makeLoggedInAdminClient()
+        await createTestOrganization(admin)
+        await makeClientWithRegisteredOrganization()
+
+        const objs = await Organization.getAll(admin, {})
+        // Because database is not cleaned up before tests, there is a lot of organizations, created by previous test runs
+        // So, just expect more organizations, than created by admin in question
+        expect(objs.length).toBeGreaterThan(1)
```

Comment:
It's better to check, that this list has another user-owned organization (prom line 43). 
Use: `const objs = await Organization.getAll(client, {}, { sortBy: ['updatedAt_DESC'] })`
```
        expect(objs).toEqual(expect.arrayContaining([
            expect.objectContaining({
                id: client43Line.organization.id,
            }),
        ]))
```

Reply:
Added in `aac0b8d3`